cherry_pick.py

```diff
@@ -0,0 +1,54 @@
+import argparse
+import json
+import subprocess
+import sys
+
+def get_cherry_picks():
+    with open('cherry-picks.json') as f:
+        return json.load(f)
+
+
+def apply(cherry_pick, dry_run=False):
+    command = [
+        "git",
+        "cherry-pick",
+        cherry_pick['sha'],
+    ]
+
+    if cherry_pick.get('merge', True):
+        command += ['-m', '1']
```

Comment:
my git knowledge isenough to be scared but not enough to convince myself this is fine:
- are we assured that the branch we want will always be 1
- do we ever need to worry about an octopus merge?